waku/node/waku_node.nim

```diff
@@ -1239,16 +1239,19 @@ proc mountLibp2pPing*(node: WakuNode) {.async: (raises: []).} =
 # TODO: Move this logic to PeerManager
 proc keepaliveLoop(node: WakuNode, keepalive: chronos.Duration) {.async.} =
   while node.started:
-    # Keep all connected peers alive while running
+    # Keep connected peers alive while running
+    # Each node is responsible of keeping its outgoing connections alive
     trace "Running keepalive"
 
     # First get a list of connected peer infos
-    let peers =
-      node.peerManager.peerStore.peers().filterIt(it.connectedness == Connected)
+    let outPeers = node.peerManager.connectedPeers()[1]
 
-    for peer in peers:
+    for peerId in outPeers:
       try:
-        let conn = await node.switch.dial(peer.peerId, peer.addrs, PingCodec)
+        info "calling keepAlive dial", peerId = peerId
```

Comment:
I bet this one is referred by https://github.com/waku-org/nwaku/issues/3090
```suggestion
        trace "calling keepAlive dial", peerId = peerId
```

Reply:
Yes! So that log is already on master, not sure why it's shown on the diff.

I have another PR removing that log from master, that it was never supposed to enter :))